shared/models/notification.rb

```diff
@@ -0,0 +1,66 @@
+require "securerandom"
+
+module FastlaneCI
+  # All metadata about a notification.
+  class Notification
+    # Is a UDID so we're not open to ID guessing attacks
+    #
+    # @return [String]
+    attr_accessor :id
+
+    # The relative priority of a notification
+    #
+    # @return [String]
+    attr_accessor :priority
```

Comment:
Maybe we should define priorities using an enum, kinda like:
```ruby
PRIORITIES = {
      urgent: "urgent",
      normal: "normal",
      success: "success"
    }
```
Would we want any other priorities?
I can see `urgent` being used for stuff that must be fixed (and could potentially page people)
vs `normal` which is like `this build failed`, and `success` which is reserved for success messages.